gradle/libs.versions.toml

```diff
@@ -0,0 +1,44 @@
+[versions]
+hypertrace-serviceFramework = "0.1.43"
+hypertrace-grpcUtils = "0.9.0"
+hypertrace-documentStore = "0.7.18"
+
+protoc = "3.21.2"
```

Comment:
this is an older version with vulnerabilities. we've been using 3.21.7 (although looks like 3.21.8 is out now)